includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -970,7 +986,15 @@ public function update_saved_payment_method( $payment_method, $order_id, $is_tes
 			$this->customer_service->update_payment_method_with_billing_details_from_order( $payment_method, $order );
 		} catch ( Exception $e ) {
 			// If updating the payment method fails, log the error message.
-			Logger::log( 'Error when updating saved payment method: ' . $e->getMessage() );
+			Logger::log(
```

Comment:
Not sure if these logs should be translated too... or is it used by our support to debug problems?